### ISSUE TITLE
Provide an `to_full_url` Antlers modifier

### ISSUE THREAD

**Issue by o1y** [statamic/ideas#1274]
Actually, it's a pretty simple requirement, but I couldn't find a good solution.

I have a link field where the editor can select an entry. Now, I want to use this URL, for example, as the canonical URL of the page. For this, I need to convert the relative URL into an absolute URL.

Currently, I have three options:

1. Use the `join` or `ensure_left` (or other string) modifiers in Antlers to combine the app URL with the entry's URL.
2. Use the link tag, but this doesn’t work with multi-site setups. For example, `{{ link :to="my_link_field" absolute="true" }}` will turn `/en/test` into `https://mydomain.test/en/en/test` 🫨
3. Create a custom modifier like`{{ my_link_field | to_full_url }}`.

Option 3 is what I'm requesting through this idea.

**Comment by efc** [comment]
Does the `full_urls` modifier not do what you are asking for? https://statamic.dev/modifiers/full_urls

**Comment by o1y** [comment]
No, `full_urls` only works for content variables that already include link elements with relative paths.
